_posts/2018-01-02-this-is-not-a-test.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title:  "This Is Not a Test"
+date:   2018-01-02 12:00:00 -0400
+categories: culture interviews
+author: Janice Smith
+---
+
+At forward financing how we work is something we are always trying to improve on.
+We have some core tenants to how we work as a team, our director of engineering,
+Patrick Hereford, wrote a great post about that recently. You should check it out [here](http://tech.forwardfinancing.com/engineering/culture/2017/09/30/our-engineering-culture-methodologies-and-philosophies.html){:target='_blank'}
+
+We have been in the middle of building the team’s numbers here,
+in a push that will eventually more than double the size or our group.
+These positions are for different levels of engineers and for both local and remote team members.
+This is a very exciting time for us, but how successful we are really depending on who we are looking for.
+
+What we care about it finding a team member is the interesting in joining a blameless culture that is rooted in learning as we build the product the business needs, which we mainly do with code review and pairing.
```

Comment:
`we care about is finding a team member interested in`

Reply:
Or just tighten it up: "We care about finding team members interested in joining..."

Reply:
💯